introduction.md

```diff
@@ -4,4 +4,11 @@ Hi I'm Peter and i'm following the mastering github course on codeschool.com
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?